static/src/javascripts/projects/common/modules/experiments/tests/contributions-header.js

```diff
@@ -32,7 +34,8 @@ define([
         this.idealOutcome = '';
         this.canRun = function () {
             var pageObj = window.guardian.config.page;
-            return !(pageObj.isLiveBlog || pageObj.isAdvertisementFeature) && pageObj.edition === 'UK';
+            var userCookie = cookies.get('GU_U');
```

Comment:
Oh! I didn't mean you should actually read the cookie directly- `window.guardian.config.user` probably _is_ the right thing to read, I just thought it was worth checking whether it was definitely completed before the AB javascript was run.